python/nammu/view/AtfEditArea.py

```diff
@@ -60,6 +62,41 @@ def get_line_num(self, position):
         text = self.text[0:position]
         return text.count('\n') + 1
 
+    def setText(self, text):
+        '''
+        Override JTextPane's setText to call syntax highlighting so that it
+        still works when you set text from elsewhere in the code.
+        '''
+        super(AtfEditArea, self).setText(text)
```

Comment:
Added it now. IT was being called elsewhere everytime after setText, so it's better in here.